bin/get-amy.py

```diff
@@ -70,6 +70,9 @@ def main(amy_url, output_file, tags, all_=True):
         del slugs
         del workshops
 
+    # Always sort workshops by the ascending start date.
+    unique_workshops = sorted(unique_workshops, key=lambda w: w['start'])
```

Comment:
Yes please.

On 2016-01-19 10:02 AM, Piotr Banaszkiewicz wrote:
>
> In bin/get-amy.py 
> <https://github.com/swcarpentry/website/pull/183#discussion_r50121970>:
>
> > @@ -70,6 +70,9 @@ def main(amy_url, output_file, tags, all_=True):
> >          del slugs
> >          del workshops
> >
> > +    # Always sort workshops by the ascending start date.
> > +    unique_workshops = sorted(unique_workshops, key=lambda w: w['start'])
>
> Personally I wouldn't worry about it, but if you want I can add 
> sorting by the slug as well.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/swcarpentry/website/pull/183/files#r50121970>.
>

-- 
Dr Greg Wilson
Director of Instructor Training
Software Carpentry Foundation